test/unit/lib/censorBirthV0.js

```diff
@@ -0,0 +1,275 @@
+const roleWithFullAccess = ['full-details'];
+const roleWithoutFullAccess = ['death'];
+
+const birthData = {
+  location: {
+    name: "Registration District, Subdistrict town, Adminshire",
+    administrativeArea: "Adminshire",
+    subDistrict: "Subdistrict town",
+    nameQualifier: "generated by concatenating registrationDistrict, subDistrict and administrativeArea",
+    registrationDistrict: "Registration District"
+  },
+  subjects: {
+    child: {
+      originalName: {
+        givenName: "Inconvenient",
+        surname: "Mistake",
+        fullName: "Inconvenient Mistake",
+        qualifier: "generated from prepending forename to surname"
+      },
+      name: {
+        givenName: "Expensive",
+        surname: "Mistake",
+        fullName: "Expensive Mistake",
+        qualifier: "generated from prepending forename to surname"
+      },
+      dateOfBirth: "2020-09-23",
+      sex: "Male",
+      birthplace: "Bath"
+    },
+    father: {
+      name: {
+        givenName: "FATHER FORENAME",
+        surname: "FATHER SURNAME",
+        fullName: "FATHER FORENAME FATHER SURNAME",
+        qualifier: "generated from prepending forename to surname"
+      },
+      birthplace: "BRISTOL",
+      occupation: "CLEANER"
+    },
+    mother: {
+      name: {
+        givenName: "MOTHER FORENAME",
+        surname: "MOTHER SURNAME",
+        fullName: "MOTHER FORENAME MOTHER SURNAME",
+        qualifier: "generated from prepending forename to surname"
+      },
+      birthplace: "Exeter",
+      occupation: "Teacher",
+      maidenSurname: "MAIDEN SURNAME",
+      marriageSurname: "MARRIAGE SURNAME",
+      usualAddress: "ADDRESS 1"
+    },
+    informant: {
+      name: {
+        givenName: "Informant Forename1",
+        surname: "Informant1 Surname",
+        fullName: "Informant Forename1 Informant1 Surname",
+        qualifier: "generated from prepending forename to surname"
+      },
+      usualAddress: "INFORMANT ADDRESS",
+      qualification: "INFORMANT QUALIFICATION",
+      signature: "INFORMANT SIGNATURE"
+    }
+  },
+  systemNumber: 12345,
+  id: 98765,
+  date: "2020-09-23",
+  registrarSignature: "Registrar signature",
+  status: {
+    potentiallyFictitiousBirth: false,
+    correction: "None",
+    cancelled: false,
+    blockedRegistration: false,
+    marginalNote: "Marginal Note",
+    reRegistered: false
+  },
+  previousRegistration: {
+    date: "2020-05-23",
+    systemNumber: 3455667
+  }
+}
+
+const redactedBirthData = {
+  location: {
+    name: "Registration District, Subdistrict town, Adminshire",
+    administrativeArea: "Adminshire",
+    subDistrict: "Subdistrict town",
+    nameQualifier: "generated by concatenating registrationDistrict, subDistrict and administrativeArea",
+    registrationDistrict: "Registration District"
+  },
+  subjects: {
+    child: {
+      originalName: {
+        givenName: null,
+        surname: null,
+        fullName: null,
+        qualifier: null
+      },
```

Comment:
I wonder if this should be included.

I have some vague memory of searching against both names.

Reply:
In the v0 view the original name is referenced at all which is why it's been redacted. Happy to change but there's nothing (bar your comment) indicating that this should be the case.

Reply:
Looks like we don't search both names and we don't include the original name.
So I think you're implementation is correct.